2-mandatory/5-eligible-students.js

```diff
@@ -7,7 +7,15 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
 */
 
-function getEligibleStudents() {}
+function getEligibleStudents(nameAndCountArray) {
+  let eligibleStudents = [];
+  nameAndCountArray.map(mark => {
```

Comment:
map will return a new array where all elements have had a transformation applied to them consider using a filter here how could it make your code simpler.